src/components/shared/Description.tsx

```diff
@@ -1,18 +1,19 @@
 import { MarkdownViewer } from '@stoplight/markdown-viewer';
 import { Popover } from '@stoplight/ui-kit';
+import { Box } from '@stoplight/mosaic';
 import * as React from 'react';
 
 export const Description: React.FunctionComponent<{ value: string }> = ({ value }) => (
   <Popover
     boundary="window"
     interactionKind="hover"
-    className="ml-2 flex-1 truncate flex items-baseline"
-    target={<div className="text-darken-7 dark:text-lighten-7 w-full truncate">{value}</div>}
+    className="flex-1 truncate flex items-baseline"
```

Comment:
All of the tailwind classes are now prefixed under `sl-`. If we really have to use them (in this case we do until can replace with new popover), we should start using the prefixed classes because these non prefixed tailwind classes will go away when ui-kit is removed.